package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "eslint 'src/**/*.ts'",
     "lint:fix": "npm run lint -- --fix",
     "watch": "tsc -p ./ && rollup -c -w",
-    "prepublishOnly": "tsc -p ./ && rollup -c"
+    "prepare": "npm run-script build"
```

Comment:
Sorry just a quick change request: would you mind switching this to `npm run` instead of `npm run-script`, just to stay consistent with the existing commands?

Reply:
No problem! ✅ Done